mllm/backends/qnn/QNNUtils.hpp

```
@@ -89,6 +88,8 @@ bool freeQnnTensor(Qnn_Tensor_t& tensor);
 
 bool freeQnnTensors(Qnn_Tensor_t*& tensors, uint32_t numTensors);
 
+void __printQnnTensor(const Qnn_Tensor_t* tensor);  // for debug use
```

Clang (14.0.6)

[error] declaration uses identifier '__printQnnTensor', which is a reserved identifier

(bugprone-reserved-identifier,-warnings-as-errors)